src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Ios/CameraPreviewControlRenderer.cs

```diff
@@ -316,5 +330,27 @@ private IReadOnlyList<CameraFocusMode> RetrieveSupportedFocusModes()
 
             return supportedFocusModes;
         }
+
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+        private async Task<IReadOnlyList<ICamera>> RetrieveCamerasAsync()
+#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
+        {
+            var cameras = new List<ICamera>();
+            var devices = AVCaptureDevice.DevicesWithMediaType(AVMediaType.Video);
+            if (devices != null)
+            {
+                foreach (var device in devices)
```

Comment:
LINQ?

Reply:
Unless i'm missing something, I don't see much benefit of using LINQ for what we're doing so feels like a personal preference. Doesn't take advantage of deferred execution and add soverhead. FYI, BuildIt currently has a lot of similar code as well...